pennylane_lightning/src/rework/Apply.cpp

```diff
@@ -11,43 +11,16 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-#include <set>
-
 #include "Apply.hpp"
 #include "Gates.hpp"
 #include "StateVector.hpp"
 #include "Util.hpp"
+#include "Util.cpp"
```

Comment:
Do we have to include this? I couldn't work out how not to :thinking:

Reply:
I don't think we have to include the cpp file here, but we do have to add it to the compilation step.  

I think....

Reply:
If we have to include the cpp file here then something is very wrong indeed. What's wrong with just including the header file?

Reply:
Yep, agree that this should not be included. If there are undefined references at the linking step, that means that the `.cpp` file was not compiled and the linker cannot find object files that were the result of compilation. So `Util.cpp` would have to be passed with other sources to the compiler e.g., `g++ main.cpp Util.cpp` for a basic command that would use two sources and no external libraries.

Reply:
Removed the `Util.cpp`, the solution was that I forgot to update `setup.py`.